examples/nlp/question_answering/question_answering_squad.py

```diff
@@ -31,9 +31,9 @@ def main(cfg: DictConfig) -> None:
     log_dir = exp_manager(trainer, cfg.get("exp_manager", None))
     infer_datasets = [cfg.model.validation_ds, cfg.model.test_ds]
     for infer_dataset in infer_datasets:
-        if infer_dataset.output_prediction_file is not None:
+        if infer_dataset.get('output_prediction_file', None) is not None:
```

Comment:
So generally I am for dropping all of those param checks in favor of Structured Configs with default values.

Thanks to this the user will have those params in one place, instead of going through different model/datasset methods trying to figure out where they are set...

But it is up to you whether you want to follow this approach.

Reply:
done